src/appleseed/renderer/kernel/rendering/final/texturecontrolledpixelrenderer.h

```diff
@@ -0,0 +1,89 @@
+
+//
+// This source file is part of appleseed.
+// Visit https://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2010-2013 Francois Beaune, Jupiter Jazz Limited
```

Comment:
Replace those two lines by a single line:

    Copyright (c) 2019 Stephen Agyemang, The appleseedhq Organization